quickstart/oneclick_instruction_box.md

```diff
@@ -0,0 +1,14 @@
+### Prerequisites
+1. Set up Java on your machine. For instructions, consult a Java environment setup tutorial, like [this one](https://www.tutorialspoint.com/java/java_environment_setup.htm).

```

Comment:
Maybe we should recommend openjdk over oracles?

Reply:
Interesting idea. I do often prefer open source, especially considering that we're **in** an open source project. But I don't think there's any **practical** difference between them, and my impression is that officially Oracle is considered the reference version of Java.

Reply:
The site is very outdated (it recommends notepad as an editor but not intellij).
And Oracle requires you to create an account to download JDK 8, and I think it installs bloatware by default.

Check out https://adoptium.net/?variant=openjdk8

Reply:
What bloatware does the Oracle JDK install? That's the version I use, and I don't remember it including any such additional software.

Do you think it's relevant that Oracle's JDK is the official reference version?
I think Oracle also supports its JDKs for much longer than OpenJDK, so my impression is that it's **preferred in the enterprise**.
Oracle's version also includes improvements that can't be included in OpenJDK because they aren't open source.

I don't think we're in the business of trying to find them the most up-to-date tutorial for setting up their local Java env. It's a bit of a bonus that we even provide an example of one.

Reply:
I got lost in this discussion, so forgot to check whether we're even **recommending** which JDK to use. We're not, as far as I remember and see in the README.

I think this is a valuable discussion to decide which JDK to recommend if we were recommending one. But since we don't, and conceptually which JDK they use is very much out of scope of this Quickstart, I don't think this discussion should hold up releasing this Qucikstart which the PMs are under a tight schedule to release.

Reply:
Fair enough, I'll resolve